lib/series.js

```diff
@@ -71,7 +71,10 @@ module.exports = {
             return String(eventData.labels);
         },
         host: Os.hostname,
-        pid: 'pid'
+        pid: 'pid',
+        remoteIp: function (eventData) {
+            return String(eventData.source && eventData.source.remoteAddress);
```

Comment:
It just gets serialized to json, right? So undefined properties are omitted during `JSON.stringify` and (I assume) influx is none the wiser.